Tests/LoggingTests/LoggingTest.swift

```diff
@@ -711,14 +775,14 @@ class LoggingTest: XCTestCase {
 
         var logger = Logger(label: "test")
         logger.logLevel = .error
-        logger.error(Dummy())
+        logger.error(error: Dummy())
 
         logging.history.assertExist(level: .error, message: "errorDescription")
     }
 }
 
 extension Logger {
-    public func error(_ error: Error,
+    public func error(error: Error,
```

Comment:
fyi: I had to rename this so `let error = logger.error(_:metadata:file:function:line:)` wasn't ambiguous